### ISSUE TITLE
Can this be used to monitor vault agents?

### ISSUE THREAD

**Issue by zwiebelspaetzle** [madrisan/hashicorp-vault-monitor#14]
We're using this to monitor vault servers.  Can it also monitor vault agent processes on client machines?

Just thought I'd check here before asking IT to try it.

**Comment by madrisan** [comment]
Yes you can. `hashicorp-vault-monitor` checks if the vault service or cluster is up, running and in the unsealed state. If not an error message is raised.

**Comment by madrisan** [comment]
Closed for inactivity.
